### PR TITLE
ERM-1735, ERM-1736 keyboard shortcuts in ui-dashboard

### PR DESCRIPTION
**Pull request by CalamityC**
* add keyboard shortcuts, refs ERM-1735
* display keyboard shortcuts modal, refs ERM-1736
* remove lazy from index, refs ERM-1735, ERM-1736
* keep Suspense, removal is causing another error
* edit shortcut is not covered here

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -21,16 +36,61 @@ const App = (appProps) => {
...
+        >
+          <AppContextMenu>
+            {(_handleToggle) => (
+              <NavList>
+                <NavListSection>
+                  <NavListItem
+                    id="keyboard-shortcuts-item"
+                    onClick={() => { setIsShortcutsModalOpen(true); }}
+                  >
+                    <FormattedMessage id="ui-agreements.appMenu.keyboardShortcuts" />
```
Comment (aditya-matukumalli) [inline src/index.js:70]: Needs a new translation key in `ui-dashboard` translations

File: src/components/WidgetForm/WidgetForm.js [hunk 2]
```diff
@@ -161,38 +180,40 @@ const WidgetForm = ({
...
-        buttonStyle="danger"
-        confirmLabel={<FormattedMessage id="ui-dashboard.widgetForm.changeDefinitionWarningModal.continue" />}
-        data-test-delete-confirmation-modal
-        heading={<FormattedMessage id="ui-dashboard.widgetForm.changeDefinitionWarningModal.heading" />}
-        id="wipe-widget-form-confirmation"
-        message={<FormattedMessage id="ui-dashboard.widgetForm.changeDefinitionWarningModal.message" />}
-        onCancel={() => {
+          </Pane>
+        </Paneset>
+
```
Comment (aditya-matukumalli) [inline src/components/WidgetForm/WidgetForm.js:198]: Whitespace isn't necessary here

**Review by aditya-matukumalli — CHANGES_REQUESTED** [review]
Merge conflicts need resolutions.

File: src/index.js [hunk 3]
```diff
@@ -21,16 +36,61 @@ const App = (appProps) => {
...
+                    id="keyboard-shortcuts-item"
+                    onClick={() => { setIsShortcutsModalOpen(true); }}
+                  >
+                    <FormattedMessage id="ui-dashboard.appMenu.keyboardShortcuts" />
+                  </NavListItem>
+                </NavListSection>
+              </NavList>
+        )}
+          </AppContextMenu>
+          <Suspense fallback={null}>
```
Comment (aditya-matukumalli) [inline src/index.js:76]: @CalamityC Now that we have moved away from using `React.lazy`, we should move the usage of `Suspense` out as well, as it serves no purpose here. The routes imports are not dynmamic anymore.

File: src/index.js [hunk 4]
```diff
@@ -1,18 +1,33 @@
...
 
 import PropTypes from 'prop-types';
 import Registry from './Registry';
 
+import DashboardsRoute from './routes/DashboardsRoute';
+import DashboardRoute from './routes/DashboardRoute';
+import DashboardOrderRoute from './routes/DashboardOrderRoute';
+import WidgetCreateRoute from './routes/WidgetCreateRoute';
+
 const Settings = lazy(() => import('./settings'));
```
Comment (aditya-matukumalli) [inline src/index.js:25]: Any reason this is left out as a dynamic imoprt?

**Review by aditya-matukumalli — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 4 above)
Comment (CalamityC) [reply src/index.js:25]: it's not causing issues as the others, so I left it as it was

File: src/index.js (hunk 3 above)
Comment (CalamityC) [reply src/index.js:76]: It's giving me another error when I remove the `<Suspense fallback={null}>`, see my comments above

> * keep Suspense, removal is causing another error

File: src/index.js (hunk 4 above)
Comment (aditya-matukumalli) [reply src/index.js:25]: Yeah, I mean just for consistency lets take it out. We can add it back when we actually have code splitting fully supported. Refer to the comment here https://folio-project.slack.com/archives/CAN13SWBF/p1580423284014600 for more information.

**Review by aditya-matukumalli — APPROVED** [review]

**Pull request merged**